Add `Tournaments/MALT` (#10196)
ppy/osu-wiki: wiki/Tournaments/MALT/1/en.md

| **Holo the Wise** ::{ flag=CA }:: | **4** | 0 | ::{ flag=US }:: Mables | [#1](https://osu.ppy.sh/community/matches/110305062) |
| Evaluation ::{ flag=US }:: | 1 | **4** | ::{ flag=FR }:: **poketoine** | *win by default* |

### Qualifiers

The results of the Qualifiers can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/1WZMBWnq-Mv-n791tDTASkQQZNsHOpgP3xCn98vBZdm4/edit?rm=minimal).

## Ruleset

### General Rules

- Players who are ranked #1,000 to #40,000 in the osu!mania 7K-only rankings are the only ones allowed to join the tournament. Player–staff eligibility rules for officially supported tournaments also apply.
- The scoring system used is ScoreV2.
- The following mods are allowed: Fade In (FI), Hidden (HD), Flashlight (FL), Mirror (MR), No Fail (NF).

### Round Overview

#### Qualifier Stage

- Mappools are constructed to have 5 maps: 2 rice-based, 2 LN-based, and 1 hybrid.
- Players are scheduled to a multiplayer room according to their availability.
- Each player has 2 attempts at each map, where only the best attempt for each map will count.

> - Each map has its own leaderboard, and each player's overall seed is determined by the average of these ranks. The top 48 players of the Qualifiers proceeds to the next rounds.

Reviewer: ```suggestion
- Each map has its own leaderboard, and each player's overall seed is determined by the average of these ranks. The top 48 players of the Qualifiers proceed to the next rounds.
```

- The top 16 players immediately advance to the bracket stage. Players seeded from 17 to 48 had to participate in the playoffs.

#### Playoffs

> - Players participating in this round are to face each other in a one-on-one best-of-7 match.
> - Matchups are determined by this formula: ``65 - seed``. (This means that seed 17 faces seed 48, 18 vs 47, and so on.)
> - The mappool size is determined as follows: 4 rice-based, 2 LN-based, 3 hybrids, 1 tiebreaker.
> - Winners from the playoffs take the final 16 slots in the bracket stage. The losers are eliminated.

Reviewer: ```suggestion
- Players participating in this round are to face each other in a one-on-one best-of-7 match.
- Matchups are determined by this formula: `65 - seed`. (This means that seed 17 faces seed 48, 18 vs. 47, and so on.)
- The mappool size is determined as follows: 4 rice-based, 2 LN-based, 3 hybrids, 1 tiebreaker.
- Winners from the playoffs take the final 16 slots in the bracket stage. The losers are eliminated.
```

#### Bracket stage

> - Players are seeded into a double-elimination bracket. They comprise of the top 16 from the Qualifiers and the winners from the playoffs to top up the remaining 16 slots.
> - Mappool sizes and best-of format for each round are given as follows:
>   - Round of 32 (best-of-7) - 4 rice-based, 2 LN-based, 3 hybrids, 1 tiebreaker.
>   - Round of 16 (best-of-9) - 5 rice-based, 3 LN-based, 3 hybrids, 1 tiebreaker.
>   - Quarterfinals and Semifinals (best-of-11)- 6 rice-based, 4 LN-based, 3 hybrids, 1 tiebreaker.
>   - Finals and Grand Finals (best-of-13) - 7 rice-based, 5 LN-based, 3 hybrids, 1 tiebreaker.

Reviewer: ```suggestion
- Players are seeded into a double-elimination bracket. They comprise of the top 16 from the Qualifiers and the winners from the playoffs to top up the remaining 16 slots.
- Mappool sizes and best-of format for each round are given as follows:
  - Round of 32 (best of 7): 4 rice-based, 2 LN-based, 3 hybrids, 1 tiebreaker.
  - Round of 16 (best of 9): 5 rice-based, 3 LN-based, 3 hybrids, 1 tiebreaker.
  - Quarterfinals and Semifinals (best of 11): 6 rice-based, 4 LN-based, 3 hybrids, 1 tiebreaker.
  - Finals and Grand Finals (best of 13): 7 rice-based, 5 LN-based, 3 hybrids, 1 tiebreaker.
```

### Match Rules

- The multiplayer room will be created and invites sent to players 5-10 minutes prior to the match start time.
- If a player is more than 10 minutes late to the for their match, the player inside the MP room will be declared the winner. If neither player appears, the player with the higher seed will be considered the winner.
- Each player will be allowed to select 1 map as a warmup. Warmups must be uploaded to the osu! servers and cannot exceed 4 minutes in length.
- Players are required to choose to ban 1 map from the mappool (excluding the tiebreaker). The pick/ban order is determined by the !roll command. The player with the lower roll bans first and the player with the higher roll picks first. Players will alternate picks until a match winner is decided.
- If both players are tied on match point (e.g. 3-3 score in a Best-of-7), the players will play the tiebreaker map of the pool to determine the winner.
- Players are given 2 minutes to ban/pick maps when it is their turn. Failure to do so means that the referee will ban/pick at random in behalf of the player.

> - Players are eligible to use a 2 minute tactical time out once per match. They are only allowed to use it when it is their turn to pick. It will take into effect after the pick timer has expired.

Reviewer: ```suggestion
- Players are eligible to use a 2-minute tactical timeout once per match. They are only allowed to use it when it is their turn to pick. It will take into effect after the pick timer has expired.
```
